### PR TITLE
ci: run vint on Vim source

### PR DESCRIPTION
**Pull request by blueyed**
Fixes https://github.com/vim-jp/vim-vimlparser/issues/111.

Alternative to https://github.com/vim-jp/vim-vimlparser/pull/136 - mainly for testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,32 +5,18 @@ git:
...
-    - name: Vim v8
-      env:
-      - VIM_VERSION=v8.0.0000
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-v8.0.txt
-    - name: Vim master
-      env:
-      - VIM_VERSION=master
-      - MAKE_TARGET=test
-      - TEST_PROFILE=vim-profile-master.txt
```
Comment (tyru) [inline .travis.yml]: why did you remove these tests?

Comment (blueyed) [reply .travis.yml]: Will be reverted - only to have a single (relevant) job here.

File: .travis.yml [hunk 2]
```diff
@@ -5,32 +5,18 @@ git:
...
     - name: Installed Vim with checks
       env:
       - VIM_VERSION=installed
-      - MAKE_TARGET="clean_compiled check checkpy js/test py/test test/node_position/test_position.out"
+      - MAKE_TARGET="checkvim"
```
Comment (tyru) [inline .travis.yml]: why did you remove `clean_compiled check checkpy js/test py/test test/node_position/test_position.out` ?

File: .travis.yml (hunk 1 above)
Comment (tyru) [reply .travis.yml]: okay 👍

File: .travis.yml (hunk 2 above)
Comment (blueyed) [reply .travis.yml]: It was also temporary to get faster feedback on CI.. :)

**Comment by blueyed** [comment]
Squashed - ready for real review.. :)
Expected to fail of course.

**Comment by tyru** [comment]
thanks 🎉
